Library/Homebrew/dev-cmd/tests.rb

```diff
@@ -37,7 +37,7 @@ def tests
 
       Homebrew.install_gem_setup_path! "bundler"
       unless quiet_system("bundle", "check")
-        system "bundle", "install", "--path", "vendor/bundle"
+        system "bundle", "install"
```

Comment:
Indeed and conversely user global configs will override this unless specified. Better to leave as-is, I think.